File: plugins/slopocop-code/skills/code-slop/scripts/tests/fixtures/clean.go
package fixtures

import "strconv"

// Sum parses each element as an integer and returns the total.
func Sum(values []string) (int, error) {
	total := 0
	for _, v := range values {
		n, err := strconv.Atoi(v)
		if err != nil {
			return 0, err
		}
		total += n
	}
	return total, nil
}

File: plugins/slopocop-code/skills/code-slop/scripts/tests/fixtures/clean.rs
/// Parses each element as an integer and returns the total.
fn sum(values: &[&str]) -> Result<i64, std::num::ParseIntError> {
    let mut total = 0;
    for value in values {
        match value.parse::<i64>() {
            Ok(n) => total += n,
            Err(e) => return Err(e),
        }
    }
    Ok(total)
}
